clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/DestroyServiceAtomicOperation.java

```diff
@@ -80,4 +101,85 @@ public Void operate(List priorOutputs) {
     return null;
   }
 
+  private void deleteMetrics() {
```

Comment:
I'd suggest moving such methods to another class so that we don't have a god class in here

Reply:
Valid point.

I want to refactor CreateServiceAtomicOperation as well... that will all happen when we get to doing the AtomicOperations.